app/models/aliases/miq_provision_ibm_cloud_vpc_workflow.rb

```diff
@@ -0,0 +1 @@
+::MiqProvisionIbmCloudVpcWorkflow = ::ManageIQ::Providers::IbmCloud::VPC::CloudManager::ProvisionWorkflow
```

Comment:
The UI calls `MiqProvisionWorkflow.class_from_platform` which does `.classify` on the catalog type.

Reply:
interesting...might be worth it to start killing that (not for this PR)

Reply:
But it also checks that `klass.name.to_s == class_name` which doesn't work for aliases :cry:

Reply:
> interesting...might be worth it to start killing that (not for this PR)

Yeah, https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/application_controller/miq_request_methods.rb#L972-L1039 is kind of a mess